src/app/modules/shared_models/section_model.nim

```diff
@@ -399,12 +383,13 @@ QtObject:
           "notificationsCount": item.notificationsCount,
           "active": item.active,
           "enabled": item.enabled,
-          "joined": item.joined,
+          "joined": (item.joined and not item.amIBanned),
```

Comment:
`item.joined` should be set to false from the module. The module will get a call from the controller, who would have an event called something like `bannedFromCommunity`. I checked the community service and it seems that we don,t even check for that yet, so I'm guessing the user doesn't see the ban state until he restarts the app. If so, it needs to be fixed.

Reply:
So @jrainville ... in general, `joined` must be set in `db` with community update message, cos ban include two steps:
- Remove member from org (which means, `joined` must be `false`). By default, we not processed it in main module and imho, the community should be `spectated`, just for save consistent state in ui. "Ban" pill should be displayed if `spectated & banned` 
- Add member to `banList`

What do you think? Cos now, when user restart the app, the community disappears (cos it's not joined and not spectated)

Reply:
@osmaczko wdyt? :)

Reply:
If I understood correctly, then when the signal that you are banned is received, `joined` will be set to `false` and `spectate` to `true`. Then, the community will still appear, but with the "banned pill".
This above should be true for on signal and for when restarting the app